packages/lit-dev-content/site/tutorials/content/working-with-lists/03.md

```diff
@@ -0,0 +1,29 @@
+The rendering methods we've seen so far make it so that Lit keeps track of the
+DOM elements and will update them accordingly when the property items change.
+This is usually fine and efficient but sometimes we may want to control how
+the each item is tied to the DOM element so that we'd rather have the DOM
+element move rather than updating.
+
+For instance, take a look at the example component we have here where each
+todo item is rendered as a list item with a checkbox. We also have buttons
+to sort these items in ascending and descending alphabetical order. Try
+checking one of the items and changing the sort order. You'll notice that
+the item text changes but not the position of the checked box. This is
+because Lit is updating the text rather than moving the entire DOM node.
+
+This is where the [`repeat`](https://lit.dev/docs/templates/directives/#repeat)
+directive comes in. We can replace the expression that's currently using
+the `map` method to utilize `repeat` like this.
+```ts
+import {repeat} from 'lit/directives/repeat.js';
+```
+```ts
+${repeat(
+  this.items,
+  (item) => item.id,
+  (item) => html`<li><label><input type="checkbox">${item.task}</label></li>`
+)}
+```
+
+Check out the [documentation](https://lit.dev/docs/templates/lists/#when-to-use-map-or-repeat)
+on when to use `map` or `repeat` for more details!
```

Comment:
Maybe add a "congrats you're done" step that tells them what more to read up on next like the templating docs or a topic that you think would logically follow lists and arrays like calling `requestUpdate` or `hasUpdated` or how Lit handles reactive properties changes.

Reply:
oh and add a recap in that section